packages/app-desktop/gui/MainScreen/commands/moveToFolder.test.js

```diff
@@ -0,0 +1,106 @@
+const moveToFolder = require('./moveToFolder');
+
```

Comment:
All new files should be TypeScript

Reply:
Also unfortunately I have no idea what these test units do. Have a look at what other test units do. For example the title is often `it('should do this or that')`. What does your code do? Explain it clearly in the test title.

Reply:
> Also unfortunately I have no idea what these test units do

It tests the `AddOptions` function that returns paths of folders with different scenarios.

Reply:
> Have a look at what other test units do. For example the title is often `it('should do this or that')`. What does your code do? Explain it clearly in the test title.

OK, I will try to refactor the code and convert the unit test to TS.